### DISCUSSION TITLE
Firefox bookmarks etc missing after uninstall

### DISCUSSION THREAD

**Discussion by michaepr** [netblue30/firejail#4792] [Q&A] [answered]
Hello,

I have just uninstalled firejail after using it for many years on my Arch Linux system. On restart my firefox profile (bookmarks, etc) has disappeared. I believe I now have the profile I used before installing firejail in the first place. Is there a way I can get back the profile with all my data? If it was deleted during the uninstall I could use a semi-recent backup where firejail was still installed.

Thanks for your help!

**Comment by rusty-snake** [comment, marked as answer]
It sound like you used something like `private ${HOME}/my-private-firefox-home`. If that's the case, you can use `firefox --profile ${HOME}/my-private-firefox-home/.mozilla/firefox/PROFILE` (but create copy of the profile first, just to be safe).

**Reply by michaepr** [reply]
Worked perfectly, thank you very much!
